animareflection/ui
# Update `Tooltip` component

## Description

##### Task link: https://trello.com/c/vIGkps7p/253-tooltip-patches

Updated TooltipPositioner to include a `zIndex`.

## Test Steps

1. Verify that the `zIndex="menu"` will work for all use cases.
2. Verify that the `Tooltip` component is indexed properly when added to the downstream `Menu` component.
